src/components/controllers/doc-page.jsx

```diff
@@ -53,9 +51,8 @@ export function DocLayout() {
 
 function OldDocsWarning() {
 	const { name, version } = useRoute().params;
-	const { path } = useLocation();
 
-	if (!isDocPage(path) || version === LATEST_MAJOR) {
+	if (version === LATEST_MAJOR) {
```

Comment:
Was a bit of a problem as `useLocation`, upon navigating from (say) `/` -> `/guide/...` immediately updates with the new path, but as the route wasn't loaded yet, `name` & `version` would be `undefined`. This would result in the banner flashing upon nav.

We could correct that here, but it's better to conditionally render the entire component. `OldDocsWarning` should never be loaded on anything but guide pages.